file_system/repository/file_repository.py

```diff
@@ -26,7 +25,7 @@ def delete(self, id):
             raise FileNotFoundException("File with id:%s does not exist" % str(id))
 
     @classmethod
-    def filter(cls, queryset=None, path=None, path_in=[], path_regex=None, file_type=None, file_type_in=[], file_name=None, file_name_in=[], file_name_regex=None, file_group=None, file_group_in=[], metadata={}, metadata_regex={}, q=None, values_metadata=None, values_metadata_list=[], filter_redact=False):
+    def filter(cls, queryset=None, path=None, path_in=[], path_regex=None, file_type=None, file_type_in=[], file_name=None, file_name_in=[], file_name_regex=None, file_group=None, file_group_in=[], metadata={}, metadata_regex=[], q=None, values_metadata=None, values_metadata_list=[], filter_redact=False, exclude=[], order_by=None, distinct=None, metadata_distribution=None):
```

Comment:
I'm not too familiar with this part of the code, but it seems that most File queries can be done with built-in django methods. maybe there are use cases for this function that I'm not aware of though